sphinx_gallery/tests/test_gen_rst.py

```diff
@@ -474,6 +474,121 @@ def test_empty_output_box(gallery_conf):
     assert output.isspace()
 
 
+code_repr_only = (
+    'class repr_only_class():\n'
+    '    def __init__(self):\n'
+    '        pass\n\n'
+    '    def __repr__(self):\n'
+    '        return "This is the __repr__"\n\n'
+    'class_inst = repr_only_class()\n\n'
+    'class_inst'
+)
+
+code_repr_and_html = (
```

Comment:
I tried to do this but it wouldn't work! I think it may have been an indentation issue (from some SO questions) but I am open to suggestions, esp if you have experience with this kind of thing before.